CMakeLists.txt

```diff
@@ -317,7 +322,7 @@ if(${MPI_NEEDED} MATCHES ON )
   set(CMAKE_CXX_LINK_FLAGS ${CMAKE_CXX_LINK_FLAGS} ${MPI_LINK_FLAGS})
 
   # Note: that this will be included in the compilation of the kratos core, so that it is not needed to include it somewhere else
-  include_directories(${MPI_INCLUDE_PATH})
+  include_directories(SYSTEM ${MPI_INCLUDE_PATH})
```

Comment:
My only concern is this. Is strictly necessary? I remember having problems with boost while trying to suppress warnings this way if there was already a system version of the library. For this case I am thinking mostly on clusters ( they should have MPI implementation in modules but you can never be 100% sure)

That said, I've tried in my computer and it works. I comment it anyway so if it gives problems we have a hint on this.

Reply:
I get warnings abt overrides from MPI (openMPI 1.10 in my case) if I don't do this

What exactly were the problems? Shouldn't MPI always be seen as "System", independent if it is installed by yourself or provided?

Reply:
from CMake docu:

If the SYSTEM option is given, the compiler will be told the directories are meant as system include directories on some platforms. Signalling this setting might achieve effects such as the compiler skipping warnings, or these fixed-install system files not being considered in dependency calculations - see compiler docs.

Reply:
Yep is as the doc says. but as I said, no problem right now.

Reply:
Ok thx